platform/android/export/export.cpp

```diff
@@ -2699,40 +2770,60 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		}
 
 		APKExportData ed;
-		err = export_project_files(p_preset, rename_and_store_file_in_gradle_project, &ed,
-				store_so_file_in_gradle_project);
+		err = export_project_files(p_preset, rename_and_store_file_in_gradle_project, &ed, store_so_file_in_gradle_project);
 
 		if (err != OK) {
 			EditorNode::add_io_error("Could not export project files to gradle project\n");
 			return err;
 		}
 
+		bool export_bundle = bool(p_preset->get("custom_template/export_as_bundle"));
+		if (!export_bundle) { //handles creating the _cl_ file
+			Vector<uint8_t> command_line_file;
+			err = get_command_line_file(p_preset, p_path, p_flags, &command_line_file);
+			if (err != OK) {
+				EditorNode::add_io_error("Could not write expansion package file");
+				return err;
+			}
+			store_file_in_gradle_project("res://android/build/assets/_cl_", command_line_file, 0);
+		}
+
 		OS::get_singleton()->set_environment("ANDROID_HOME", sdk_path); //set and overwrite if required
 
 		String build_command;
+		String move_command;
```

Comment:
Not used - delete?

Reply:
Can you explain why copying is better than moving? I was thinking that we leave that up to the user. However, having too many choices is also a bad thing, so I'm okay with copying every time and not moving at all

Reply:
If nothing changed in the project, gradle can presumably look at the output artifact and see that it's already up-to-date and do no work.

Having a UI config for whether to copy or move is "too many choices" as you say.

Let's go with just copying (remove the mv option).